File: Servers/Python/BlackSheep/routers/json_response.py
from blacksheep.server.responses import json
from blacksheep import Application, Response

from Servers.Python.utils.path import JsonResponse

import Servers.test_data


def config_routes(
        app: Application,
) -> None:
    @app.router.get("/1_k_json")
    async def one_k_json() -> Response:
        return json(Servers.test_data.JSON_1K)

    @app.router.get(JsonResponse.ten_k_json)
    async def ten_k_json() -> Response:
        return json(Servers.test_data.JSON_10K)

    @app.router.get(JsonResponse.hundred_k_json)
    async def hundred_k_json() -> Response:
        return json(Servers.test_data.JSON_100K)

    @app.router.get(JsonResponse.one_kk_json)
    async def one_kk_json() -> Response:
        return json(Servers.test_data.JSON_1M)

    @app.router.get(JsonResponse.five_kk_json)
    async def five_kk_json() -> Response:
        return json(Servers.test_data.JSON_5M)

File: Servers/JS/Express/routers/jsonResponse.js
const oneKJsonData = require("../../../test_data/1K.json");
const tenKJsonData = require("../../../test_data/10K.json");
const hundredJsonData = require("../../../test_data/100K.json");
const oneKKJsonData = require("../../../test_data/1M.json");
const fiveKKJsonData = require("../../../test_data/5M.json");

const router = require("express").Router();

const { jsonResponse } = require("../../utils/paths");

router.get(jsonResponse.oneKJson, async (_, res) => {
  res.send(oneKJsonData);
});
router.get(jsonResponse.tenKJson, async (_, res) => {
  res.send(tenKJsonData);
});
router.get(jsonResponse.hundredKJson, async (_, res) => {
  res.send(hundredJsonData);
});
router.get(jsonResponse.oneKKJson, async (_, res) => {
  res.send(oneKKJsonData);
});
router.get(jsonResponse.fiveKKJson, async (_, res) => {
  res.send(fiveKKJsonData);
});

module.exports = router;
